src/components/header/header.tsx

```diff
@@ -44,7 +44,6 @@ import { Component, h, Prop } from '@stencil/core';
  * @exampleComponent limel-example-header-responsive
  * @exampleComponent limel-example-header-narrow
  * @slot - Content (actions) to be put inside the far right surface of the header
- * @private
```

Comment:
Same comment as for popover:

> Even though this technically only affects the docs, I think the commit should be a `feat` since, while we are _technically_ only making the docs public, and the feature was public all along, the idea is of course that people should not have been using the feature, but now they can, so we are "adding" a feature 🙂

Reply:
fixed! 🙂